test/models/user.rb

```diff
@@ -7,4 +9,7 @@ class User
   attr_accessor :otp_secret_key, :email
 
   has_one_time_password
+  def attributes
+    { 'otp_secret_key' => otp_secret_key, 'email' => email }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.